crates/sui-rosetta/src/account.rs

```diff
@@ -10,48 +12,125 @@ use futures::StreamExt;
 use sui_sdk::rpc_types::StakeStatus;
 use sui_sdk::{SuiClient, SUI_COIN_TYPE};
 use sui_types::base_types::SuiAddress;
+use tracing::info;
 
 use crate::errors::Error;
 use crate::types::{
     AccountBalanceRequest, AccountBalanceResponse, AccountCoinsRequest, AccountCoinsResponse,
-    Amount, Coin, SubAccount, SubAccountType, SubBalance,
+    Amount, BlockIdentifier, Coin, SubAccount, SubAccountType, SubBalance,
 };
 use crate::{OnlineServerContext, SuiEnv};
+use std::time::Duration;
 
 /// Get an array of all AccountBalances for an AccountIdentifier and the BlockIdentifier
 /// at which the balance lookup was performed.
 /// [Rosetta API Spec](https://www.rosetta-api.org/docs/AccountApi.html#accountbalance)
-pub async fn balance(
+pub async fn balance_new(
```

Comment:
Whats the motivation behind changing the name of this function?

Reply:
oh the reason we changed was since we wanted to test the new one without changing existing. @caogc since we are not using the other one we should just make it `balance`

Reply:
done